locations/spiders/bipa_hr.py

```diff
@@ -0,0 +1,35 @@
+from scrapy import Spider
+from scrapy.http import JsonRequest, Response
+
+from locations.hours import DAYS_SR, OpeningHours
+from locations.items import Feature
+
+
+class BipaHRSpider(Spider):
+    name = "bipa_hr"
+    item_attributes = {"brand": "Bipa", "brand_wikidata": "Q864933"}
+    custom_settings = {
+        "DEFAULT_REQUEST_HEADERS": {
+            "ModuleId": "841",
+            "TabId": "195",
+        }
+    }
```

Comment:
Instead of setting the headers for all requests, please set headers on the `JsonRequest` in `start_requests()`